pkg/descheduler/policyconfig.go

```diff
@@ -140,33 +140,21 @@ func validateDeschedulerConfiguration(in api.DeschedulerPolicy, registry pluginr
 				pluginUtilities := registry[pluginConfig.Name]
 				if pluginUtilities.PluginArgValidator != nil {
 					err := pluginUtilities.PluginArgValidator(pluginConfig.Args)
-					errorsInProfiles = setErrorsInProfiles(err, profile.Name, errorsInProfiles)
+					if err != nil {
+						errorsInProfiles = setErrorsInProfiles(err, profile.Name, errorsInProfiles)
+					}
 				}
 			}
 		}
 	}
-	if errorsInProfiles != nil {
-		return errorsInProfiles
-	}
-	return nil
+	return errorsInProfiles
 }
 
 func setErrorsInProfiles(err error, profileName string, errorsInProfiles error) error {
```

Comment:
`setErrorsInProfiles` is used only once. Worth making it inline.

Reply:
sound good.
update it.